example/subaccount/account.js

```diff
@@ -0,0 +1,144 @@
+ import { BeansMerchantSdk } from '../../dist/sdk.js';
+        let currentSdk = null;
+        let currentStellarAccountId = null;
+        // Load existing Accounts
+        async function loadSubaccounts() {
+            const subaccountsList = document.getElementById('subaccountsList');
+            if (!currentSdk) return;
+
+            try {
+                const accounts = await currentSdk.getMerchantAccounts();
+                subaccountsList.innerHTML = accounts.map(account => `
+                    <div class="subaccount-card">
```

Comment:
Please search and replace `subaccount` to `account`
```suggestion
                    <div class="account-card">
```